packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageList.tests.js

```diff
@@ -366,6 +366,62 @@ QUnit.module('MessageList', moduleConfig, () => {
             assert.strictEqual($dayHeaders.length, 1, 'day header is not removed after invalidate');
         });
 
+        [{
+            timestamp: new Date(),
+            dayHeaderPrefix: 'Today ',
+            scenario: 'today',
+        },
+        {
+            timestamp: new Date(Date.now() - MS_IN_DAY),
+            dayHeaderPrefix: 'Yesterday ',
+            scenario: 'yesterday',
+        }, {
+            timestamp: new Date('10.10.2024'),
+            dayHeaderPrefix: '',
+            scenario: '10.10.2024',
+        }].forEach(({ timestamp, dayHeaderPrefix, scenario }) => {
+            QUnit.test(`Day header should be formatted when dayHeaderFormat is specified on init (timestamp=${scenario})`, function(assert) {
+                const items = [{
+                    timestamp,
+                    text: 'A',
+                }];
+
+                this.reinit({
+                    items,
+                    dayHeaderFormat: 'dd of MMMM, yyyy',
+                });
+
+                const $dayHeaders = this.getDayHeaders();
+                const expectedMonth = timestamp.toLocaleString(undefined, { month: 'long' });
+                const expectedYear = timestamp.getFullYear();
+                const expectedDate = timestamp.getDate();
+                const expectedDayHeaderText = `${dayHeaderPrefix}${expectedDate} of ${expectedMonth}, ${expectedYear}`;
```

Comment:
I understand that you calculate the expected value instead of using a static one because you get new Date() values every day. However, it seems better to mock new Date() using sinon.useFakeTimers() as following
```
const clock = sinon.useFakeTimers(new Date('2021/10/17'));
```